upgrading/automated_upgrades.adoc

```diff
@@ -743,6 +743,14 @@ upgrade the component.
 To upgrade an existing EFK logging stack deployment, you review your parameters
 and run the *_openshift-logging/config.yml_* playbook.
 
+[NOTE]
+
+====
+In {product-title} 3.10, the Ansible variables for the Logging, Curator, Fluentd, Kibana, Kibana Proxy, Elasticsearch, and Event Router 
```

Comment:
Is this a 3.10 PR.   3.10 have been removed from the  supported line.

Reply:
@anpingli I removed this note to avoid confusion. If the user hits this problem, the user would be upgrading from 3.9 to 3.10, not from 3.10 to 3.11, correct?
If so, @vikram-redhat should we make an exception and update the 3,10 docs with this information?

Reply:
@mburke5678 no let's leave the 3.10 docs.